server/client.py

```diff
@@ -17,6 +17,7 @@
     parser.add_argument('--version', action='version', version=str(version))
     parser.add_argument('host_port', metavar='[host:]port', nargs=1, help='default host=127.0.0.1')
     parser.add_argument('files', metavar='file', nargs='*', help='SMT files to submit. CLI mode if empty')
+    parser.add_argument('-terminate',metavar='q', help='Terminating SMTS Server from client By q')
```

Comment:
The metavar is not used. It'd be nicer to use a flag default false with action `store_true`. See smts.py lines 20-23

Reply:
Good point! pushed.